### ISSUE TITLE
WandbLogger does not seem to support model and parameter tracking

### ISSUE THREAD

**Issue by LlewellynS96** [PyTorchLightning/pytorch-lightning#3550]
I am unsure whether this issue can be fixed from PyTorch Lightning's side or whether the issue lies with Wandb.  The problem is that the WandbLogger.watch() function
https://github.com/PyTorchLightning/pytorch-lightning/blob/c46de8a3d4a18e5030dc9f07e96bb6abc7643945/pytorch_lightning/loggers/wandb.py#L136-L137
calls the watch() function in wandb/sdk/watch_run.py (on Wandb's side), which is basically commented out. 
```
def watch(self, models, criterion=None, log="gradients", log_freq=100, idx=None):
    logger.info("Watching")
    # wandb.run.watch(watch)
```
If the last line of that function is changed to `wandb.watch( models, criterion, log, log_freq, idx)` (this function is in wandb/sdk/watch_watch.py) then everything seems to work like it should. Alternatively, I think it should be possible for WandbLogger.watch() to call  `wandb.watch( models, criterion, log, log_freq, idx)` directly.

It seems @borisdayma previously worked a lot on Wandblogger, so I'll mention him.

**Comment by borisdayma** [comment]
Hi, it should have been fixed.
Can you confirm what version of wandb you are using?

**Comment by LlewellynS96** [comment]
@borisdayma sorry, and thanks. I was using version 0.10.0 (from a week ago), but the issue was indeed fixed with 0.10.1 from 2 days ago.
